scripts/vim-patch.sh

```diff
@@ -162,22 +167,17 @@ list_vim_patches() {
   # runtime patches before between 384 and 442 have already been ported
   # to Neovim as of the creation of this script.
   local vim_commits=$(cd "${VIM_SOURCE_DIR}" && \
-    hg log --removed --template='{if(startswith("Added tag", firstline(desc)),
-      "{latesttag}\n",
-      "{if(startswith(\"updated for version\", firstline(desc)),
-        \"\",
-        \"{node}\n\")}")}' -r tip:v7-4-442)
+    git log --pretty='tformat:%H|%d' v7.4.442..HEAD | awk -f ${NEOVIM_SOURCE_DIR}/scripts/vim-patch-helper.awk)
```

Comment:
Not sure if you saw https://github.com/neovim/neovim/pull/3193#issuecomment-133289081 .

What does this list look like? Is it different than `git tag --contains v7.4.442` ?